internal/apitest/runner.go

```diff
@@ -137,7 +137,14 @@ func FilterSuites(p *Project, suiteName string) error {
 // RunProject runs the tests defined in apitest.Project
 func (r *Runner) RunProject() (int, error) {
 	exitCode := 1
-	if err := tunnel.ValidateTunnel(r.TunnelService, r.Project.Sauce.Tunnel.Name, r.Project.Sauce.Tunnel.Owner, tunnel.V2AlphaFilter, false); err != nil {
+	if err := tunnel.ValidateTunnel(
```

Comment:
hmm... `tunnel.ValidateTunnel()` or `tunnel.Validate()`?

Reply:
Indeed, can be more concise!